Dockerfile

```diff
@@ -14,7 +14,7 @@ RUN apt-get update && \
     unzip -q ./activator.zip && \
     cd collins && \
     java -version 2>&1 && \
```

Comment:
this java -version check seems weird/wrong since it will ignore the rest of the command instead of aborting